src/components/common/Button/Button.js

```diff
@@ -8,7 +8,8 @@ import style from "./Button.module.css";
 
 export default function Button(props) {
   const clickHandler = () => {
-    if (props.href) window.open(props.href, "_blank");
+    if (props.doNothing) return;

```

Comment:
Added this line so that clicking on the "BECOME A MENTOR" button does not show the toast message.